frontend/case/src/app/weekly/weekly.component.html

```diff
@@ -5,10 +5,28 @@
     </div>
     <main class="col-12 col-md-9 col-lg-8 py-md-3 pl-md-5 bd-content" role="main">
       <h1>
-        This Week
+        Week overview
       </h1>
       <div class="flex justify-between">
-        <div>Planned lessons for week {{day.weekNumber}}</div>
+        <div>
+          <div class="btn btn-group">
+              <button class="btn btn-default"
+              (click)="goToPreviousWeek()" matTooltip="Go to previous week" matTooltipPosition="left">
+              <i class="fas fa-chevron-left"></i>
+            </button>
+            <mat-form-field appearance="fill">
+              <mat-label>Week {{day.weekNumber}}</mat-label>
+              <input matInput [matDatepicker]="picker" placeholder="{{day | date:'shortDate' }}" [(ngModel)]="day"
```

Comment:
Maybe the calendar week is the only information the user needs here? The date is redundant to the first item in the view. I would propose a simple circle with the calendar week number in it and the ``go to`` button aside. Maybe this could be a own component?

Reply:
problem at hand if you look for a specific date you don't know the week by hard. so I decided to provide both, week and date. picker allows to pick a date and the two buttons allow to switch to prev and next week.

Reply:
Nice solution!